runtime/runtime/src/lib.rs

```diff
@@ -1313,12 +1313,12 @@ impl Runtime {
                 target: "runtime",
                 "process_receipt",
                 receipt_id = %receipt.receipt_id,
-                node_counter = ?state_update.trie().get_trie_nodes_count(),
                 predecessor = %receipt.predecessor_id,
                 receiver = %receipt.receiver_id,
                 id = %receipt.receipt_id,
             )
             .entered();
+            tracing::trace!(target: "runtime", node_counter = ?state_update.trie().get_trie_nodes_count(), "process_receipt start");
```

Comment:
This is somewhat duplicating the span itself, so I wouldn't mind particularly much if this log statement was removed entirely, to be honest (I believe we now have proper tracing in the form of Tempo?)

Reply:
I thought the point there was to show the trie node read counts before and after the "process_receipt". In the original implementation it was done a bit strangly by setting the before count in the span and adding the after count in the log message. I made it explicity by having a "start" and "done" log lines. Now that I think of it based on your suggestsion, perhaps I'll just have a single log line with both before and after.